src/app/wallets/pay-on-chain.ts

```diff
@@ -310,37 +312,38 @@ const executePaymentViaOnChain = async ({
           return txHash
         }
 
-        let onChainTxFee = await onChainService.lookupOnChainFee({
+        let totalFee: Satoshis
+
+        const minerFee = await onChainService.lookupOnChainFee({
           txHash,
           scanDepth: ONCHAIN_SCAN_DEPTH_OUTGOING,
         })
 
-        if (onChainTxFee instanceof Error) {
-          logger.fatal({ err: onChainTxFee }, "impossible to get fee for onchain payment")
-          onChainTxFee = toSats(0)
+        if (minerFee instanceof Error) {
+          logger.error({ err: minerFee }, "impossible to get fee for onchain payment")
+          totalFee = estimatedFee
+        } else {
+          totalFee = withdrawFeeCalculator.onChainWithdrawalFee({
+            minerFee,
+            bankFee: toSats(senderAccount.withdrawFee),
```

Comment:
why is `senderAccount.withdrawFee` not already typed correctly?

Reply:
we discuss yesterday whether fees should have their own types or not. we had not reached a conclusion.

Reply:
I have removed the fee specific types